Microsoft.Toolkit.Uwp.UI.Controls.Design/Common/Constants.cs

```diff
@@ -4,12 +4,17 @@
 
 using System;
 using System.Diagnostics.CodeAnalysis;
-using System.Linq.Expressions;
 
 [assembly: SuppressMessage("General", "SWC1001:XmlDocumentationCommentShouldBeSpelledCorrectly", MessageId = "Theming", Justification = "Correct spelling")]
 
-namespace Microsoft.Toolkit.Uwp.UI.Controls.Design.Common
+namespace Microsoft.Toolkit.Uwp.UI.Controls.Design
 {
+    internal static partial class ControlTypes
+    {
+        // HACK: Don't forget to update, if the namespace changes.
+        public const string RootNamespace = "Microsoft.Toolkit.Uwp.UI.Controls";
```

Comment:
We could make this a property that gets the name from something like this:
```cs
var s = typeof(ControlTypes).Namespace;
return s.Substring(0, s.LastIndexOf("."));
```

Reply:
Unlike `Design` projects, the `DesignTools` projects (i.e. it's extensibility model) works with type names in strings rather than objects. So, there's no need to do this programmatically. But being done in the above way means less refactoring in the future, if the namespace ever changes. That's why Roslyn CodeGen is very exciting solution. Until then, we can let this be.